data/gfswind2png.py

```diff
@@ -0,0 +1,201 @@
+import os
+import pathlib
+import json
+import argparse
+import glob
+from datetime import datetime
+
+import requests
+from requests import HTTPError
+import numpy as np
```

Comment:
```suggestion
import numpy
```

I think aliasing these makes the code harder to read for no real benefit.

Reply:
Can change, this is just convention with `numpy`